kafka_utils/kafka_manual_throttle/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Yelp Inc.
```

Comment:
Not related to this change at all, but I wonder if we ever update the year on these or if we're supposed to

Reply:
I just spend 30+ min trying to look for the right answer to this. Basically, I was unable to find whether this should be 2019 (as this is technically new software), or 2016 (as this is part of kafka-utils, whose "copyright" started that year).

But it seems that this header text doesn't really matter from a legal standpoint, sooo I'd be in favour of keeping it like this. Feel free to dig more if you feel like it though 🙂

Reply:
Nah, I was just curious what the best practice is for this kind of stuff